cv.py

```diff
@@ -156,20 +152,37 @@ def run_cv(self):
 
         return angle, distance
 
-# holds rectangle properties
-class Rectangle:
-
-    def __init__(self, x, y, width, height, area):
-        self.x = x
-        self.y = y
-        self.width = width
-        self.height = height
-        self.area = area
 
+# this class defines a rotated rectantle around a countour
 class RotatedRectangle:
 
     # box is a 2d list of rectangle coordiantes
     def __init__(self, box, area, rot_angle):
         self.box = box
         self.area = area
         self.rot_angle = rot_angle
+
+        points = []
+        for coordinates in box:
+            points.append(Point(coordinates))
+
+        #sorts points based on y value
+        points.sort(key= lambda x: x.y)
+        #lowest point
+        self.point1 = points[0]
+        #second lowest point
+        self.point2 = points[1]
+        #third lowest point
+        self.point3 = points[2]
+        #highest point
+        self.point4 = points[3]
+
+        self.height = abs(self.point4.y - self.point1.y)
+        self.width = abs(self.point3.x - self.point2.x)
```

Comment:
This is dangerous, since if any point gets updated these properties will become incorrect. It is better to make get_height() and get_width() functions.